Development setup error

Hello again
Getting this error following the development setup instructions, deleting the line in render-typings.d.ts referencing the vue index.d.ts file seems to let it run without a problem. Log is attached.

> render/render-typings.d.ts(3,1): error TS6053: File 'C:/Users/Lars/lyricfier/node_modules/@types/vue/index.d.ts' not found.

Also related the readme in the folder where the file should be:
> This is a stub types definition for vuejs (https://github.com/vuejs/vue).
> vuejs provides its own type definitions, so you don't need @types/vue installed!

[npm-debug.log.txt](https://github.com/emilioastarita/lyricfier/files/721957/npm-debug.log.txt)

--- a/package.json
+++ b/package.json
@@ -48,7 +48,6 @@
     "@types/electron-json-storage": "latest",
     "@types/mocha": "latest",
     "@types/node": "4.0.30",
-    "@types/vue": "latest",
     "electron-builder": "^6.4.0",
     "electron-packager": "^7.7.0",
     "electron-prebuilt": "^1.4.10",

--- a/render/SpotifyService.ts
+++ b/render/SpotifyService.ts
@@ -18,7 +18,10 @@ export class SpotifyService {
 
 
     protected static headers() {
-        return {'Origin': 'https://open.spotify.com'};
+        return {
+            'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36', // Placeholder user-agent
+            'Origin': 'https://open.spotify.com'
+        };
     }
 
     protected url(u:string) {
@@ -30,7 +33,11 @@ export class SpotifyService {
         if (this.oAuthToken.t) {
             return cb(null, this.oAuthToken.t);
         }
-        request.get('https://open.spotify.com/token', (err, status, body) => {
+        request.get({
+            url:'https://open.spotify.com/token',
+            rejectUnauthorized: false,
+            headers: SpotifyService.headers()
+        }, (err, status, body) => {
             if (err) {
                 return cb(err);
             }

--- a/render/render-typings.d.ts
+++ b/render/render-typings.d.ts
@@ -1,5 +1,4 @@
 /// <reference path="../node_modules/@types/electron/index.d.ts" />
 /// <reference path="../node_modules/@types/node/index.d.ts" />
-/// <reference path="../node_modules/@types/vue/index.d.ts" />
 
-declare let Notification : any;
+// declare let Notification : any;

